lazy_tensor_core/lazy_tensors/str_join.h

```diff
@@ -10,6 +12,7 @@
 
 namespace lazy_tensors {
 
+
 template <typename Range>
 std::string StrJoin(const Range& range, c10::string_view separator) {
   auto b = std::begin(range);
```

Comment:
ah, excellent. one long-term TODO for the whole project is to start migrating to our native utils instead of the XLA copied ones, so we should be looking for these along the way.  I'll make the change.